core/cleaner/src/main/java/com/cognifide/aet/cleaner/configuration/CleanerSchedulerConf.java

```diff
@@ -39,9 +43,20 @@
   long DEFAULT_KEEP_N_VERSIONS_PARAM = 1L;
 
   @AttributeDefinition(
-      name = SCHEDULE_CRON,
-      description = "CRON notation of when the job is to be fired. [example: '0 0 21 ? * *' will trigger job daily at 21:00].")
-  String schedule();
+      name = EXPIRED_SCHEDULE_CRON,
+      description = "CRON notation of when the Expired Data Cleaner job is to be fired. [example: '0 0 21 ? * *' will trigger job daily at 21:00].")
+  String expiredCleanerSchedule();
+
+  @AttributeDefinition(
+      name = ORPHAN_RUN,
+      description = "Flag that says if Orphan Cleaner should be run with a given schedule.",
+      type = AttributeType.BOOLEAN)
+  boolean runOrphanCleaner() default false;
```

Comment:
Please add some example config for orphan cleaner in cfg files under `osgi-dependencies/configs/src/main/resources/`